Makefile

```diff
@@ -24,7 +24,7 @@ vet: main.go
 	go vet $<
 
 clean:
-	rm -f bivac
+	git clean -fXd -e \!vendor -e \!vendor/**/* && rm -f ./bivac
```

Comment:
Why putting the two commands on the same line? Wouldn't be better for code maintainability to split this into two different lines?